### ISSUE TITLE
Head/Controller direction set per locomotion mode

### ISSUE THREAD

**Issue by shiftyscales** [Frooxius/NeosPublic#623]
Requested by Blaze (BlazZze#1784)- a mechanism to set the "use head direction for movement" setting on a per-locomotion module basis.

In particular- they specified:
>Is it possible to have the option "Use Head Direction for movement" disabled only when in flying locomotion? I'm used to walk in the direction I'm facing but flying up and down is really tedious as I have to also look up/down all the time.
Flying with hands like when that option is disabled works fine.

They wish to be able to use controller direction while in fly locomotion, and head direction for other modes, e.g. walk/run.

**Comment by Frooxius** [comment]
Yes, but this would need to wait for the new UI, to provide detailed configuration options.
